crates/ruff_python_codegen/src/generator.rs

```diff
@@ -1381,7 +1382,13 @@ impl<'a> Generator<'a> {
         }
     }
 
-    fn unparse_formatted(&mut self, val: &Expr, conversion: ConversionFlag, spec: Option<&Expr>) {
+    fn unparse_formatted(
+        &mut self,
+        val: &Expr,
+        debug_text: &Option<DebugText>,
```

Comment:
Nit: I find `Option<&ref>` more *natural* than passing a reference to an option (except if the option is mutable).

```suggestion
        debug_text: Option<&DebugText>,
```